signac/contrib/project.py

```diff
@@ -1306,7 +1295,7 @@ def _read_cache(self):
             return cache
 
     def index(self, formats=None, depth=0,
-              skip_errors=False, include_job_document=True):
+              skip_errors=False, include_job_document=True, **kwargs):
```

Comment:
Do we actually need to support arbitrary kwargs here?

Reply:
No, that is an error. I was trying to introduce backwards compatibility here, and I think we should probably still do that in the long run, but as of right now, the `kwargs` are unnecessary.